common/tags/pom.xml

```diff
@@ -46,5 +46,24 @@
   <build>
     <outputDirectory>target/scala-${scala.binary.version}/classes</outputDirectory>
     <testOutputDirectory>target/scala-${scala.binary.version}/test-classes</testOutputDirectory>
+    <plugins>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>build-helper-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <phase>generate-sources</phase>
+            <goals><goal>add-source</goal></goals>
+            <configuration>
+              <sources>
+                <source>src/main/java</source>
```

Comment:
We moved existing test annotations under `common/tags` from `src/main` package to `src/test` package, thus those annotation imports will not work as is, since by default `src/test` will be treated as test sources directory unless we explicitly mark it as sources directory.